NetworkPkg/UefiPxeBcDxe/PxeBcDhcp4.c

```diff
@@ -1749,14 +1749,17 @@ PxeBcDhcp4Dora (
   AsciiPrint ("\n");
 
 ON_EXIT:
-  if (EFI_ERROR (Status)) {
-    Dhcp4->Stop (Dhcp4);
-    Dhcp4->Configure (Dhcp4, NULL);
-  } else {
-    ZeroMem (&Config, sizeof (EFI_DHCP4_CONFIG_DATA));
-    Dhcp4->Configure (Dhcp4, &Config);
-    Private->IsAddressOk = TRUE;
-  }
+  if (!Private->DeviceDisconnected) {

```

Comment:
Can the MU_CHANGE get moved up to encompass all our changes?

Reply:
I think it would be easier to identify if just the `if` and closing brace for the if `}` had a MU_CHANGE tag just on each line to show they are new but the content inside the block is the same.